src/openslide-decode-jpeg.c

```diff
@@ -373,16 +389,17 @@ bool _openslide_jpeg_read(const char *filename,
     return false;
   }
 
-  return jpeg_decode(f, NULL, 0, dest, false, w, h, err);
+  return jpeg_decode(f, NULL, 0, space, dest, false, w, h, err);
 }
 
 bool _openslide_jpeg_decode_buffer(const void *buf, uint32_t len,
+                                   J_COLOR_SPACE space,
```

Comment:
Let's add a new wrapper function e.g. `_openslide_jpeg_decode_buffer_colorspace()` with the extra argument, rather than changing the existing callers to pass `JCS_UNKNOWN`.  That constant feels a bit too magic, and `_openslide_jpeg_decode_buffer_gray()` already provides precedent for using a separate function.

Reply:
ok, done!